### DISCUSSION TITLE
Issue with codespaces

### DISCUSSION THREAD

**Discussion by porfanid** [community/community#64399] [Codespaces]
### Select Topic Area

Bug

### Body

I have a problem

If I go to my codespaces, I can see that there is none. However, for some reason, the code spaces storage usage is reaching the limit and I do not have any idea why.

Can anyone help me debug this?

**Comment by Hyxoo** [comment]
Codespaces storage usage is calculated based on the disk space used by all of your codespaces and prebuilds, and the duration of their existence. This means that even if you delete your codespaces, you may still have some storage usage from the previous ones. The storage usage is assessed once an hour and recalculated as GB-month, which is a time-based measurement. For example, if you have a codespace that uses 1 GB of disk space for half a month, then you have consumed 0.5 GB-month of your quota.
To debug your problem, you can try the following steps:

- Check your billing settings for GitHub Codespaces and make sure you have set a non-zero spending limit. ²
- Check your codespace configuration files and make sure you are using the appropriate virtual machine type and image for your project. You can choose from a selection of virtual machine types, from 2 cores, 8 GB RAM, and 32 GB storage, up to 32 cores, 64 GB RAM, and 128 GB storage. By default, codespaces are created from an Ubuntu Linux image that includes a selection of popular languages and tools, but you can use an image based on a Linux distribution of your choice and configure it for your particular requirements. 
- Check your organization settings for GitHub Codespaces and make sure you have chosen who owns and pays for codespaces in your organization. You can choose whether codespaces created from your organization's repositories will be user-owned or organization-owned. If they are user-owned, then they will consume your personal account quota or billing. If they are organization-owned, then they will consume your organization quota or billing. 
- Check your prebuilds and make sure you are not creating unnecessary or outdated prebuilds that consume your storage quota. Prebuilds are snapshots of your codespace that allow you to start working faster. You can configure prebuilds to run automatically when you push changes to a branch or pull request, or manually trigger them from the GitHub UI or CLI. You can also delete prebuilds that you no longer need from the GitHub UI or CLI.

**Comment by mountainash** [comment]
This is a similar issue to #39133
